test/todo-app/.npmrc

```diff
@@ -1 +1 @@
-package-lock=false
\ No newline at end of file
+package-lock=true
```

Comment:
Eventually we should just delete the `.npmrc`s.

Reply:
Agreed - for posterity (aka future me), the reason we kept them was that there is a `test/.npmrc` file that has `package-lock=false` in it. To keep the number of changes minimal, we overrode the value in the subdirectory to avoid side effects on other projects